web/viewer.css

```diff
@@ -181,6 +182,8 @@
     --toolbar-icon-opacity: 1;
     --toolbar-icon-bg-color: ButtonText;
     --toolbar-icon-hover-bg-color: ButtonFace;
+    --toggled-hover-active-btn-color: ButtonText;
+    --toggled-hover-btn-outline: 3px solid ButtonBorder;
```

Comment:
Will the buttons still "fit" vertically inside of the toolbar with such a thick outline?

Reply:
With 2px it fits.